Add list of values and defaults for wootricSettings (#86)
Wootric/slate: source/includes/_install_js_intro.md

already set up to show the survey immediately for testing purposes only.

## The Wootric Settings Object:

```javascript
window.wootricSettings = {
  account_token: 'NPS-ABCD1234',
  aria: true,
  created_at: '1528416000',
  email: 'nps@example.com',
  facebook_page: 'https://www.facebook.com/test',
  language: 'EN'
  product_name: 'your product name',
  properties : { country: 'MX' },
  twitter_account: 'test',
  wootric_close_position: 'left',
  wootric_recommend_target: 'your audience',
};
```

This is a list of the values you can set into your wootricSettings object and it's defaults.
Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.

Variable | Default | Description
---- | ---- | -------
account_token | null | Your account token. Mandatory
aria | null |Enable extended aria support for our survey modal
created_at | null | The created_at date in your app for the end_user you are attempting to survey
email | null | The email of the end_user you are attempting to survey
facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
language | null | The language code of the survey

> product_name | depends on the language | Your product name. 

Reviewer: Since we are making an effort, it's better to show what it affects the survey with an example IMO.

Author: Great idea, I'll make sure I add an image of what this generates

Author: <img width="705" alt="example-one" src="https://user-images.githubusercontent.com/316711/47119535-0d604700-d220-11e8-8111-1d980d6866e6.png">
<img width="699" alt="example-two" src="https://user-images.githubusercontent.com/316711/47119536-0d604700-d220-11e8-88f9-61ce4001c17e.png">

properties  | null | The custom properties of the end_user you are attempting to survey
twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment
wootric_close_position | left | The position of the close button of your survey

> wootric_recommend_target | depens of the language| Your audience

Reviewer: ```suggestion
wootric_recommend_target | depends on the language| Your audience
```

You can easily install Wootric in your development environment for testing. The snippet is
already set up to show the survey immediately for testing purposes only.

## The Wootric Settings Object:

```javascript
window.wootricSettings = {
  account_token: 'NPS-ABCD1234',
  aria: true,
  created_at: '1528416000',
  email: 'nps@example.com',
  facebook_page: 'https://www.facebook.com/test',
  language: 'EN',
  product_name: 'your product name',
  properties : { country: 'MX' },
  twitter_account: 'test',
  wootric_close_position: 'left',
  wootric_recommend_target: 'your audience'
};
```

> This is a list of the values you can set into your wootricSettings object and it's defaults.

Reviewer 2: ```suggestion
This is a list of the values you can set into your wootricSettings object and its defaults.
```

Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.

Variable | Default | Description
---- | ---- | -------

> account_token | null | Your account token. Mandatory

Reviewer 2: ```suggestion
account_token | null | Your account token. Required.
```

> aria | null |Enable extended accessibility features (high contrast and extended aria support)

Reviewer 2: ```suggestion
aria | null | Enable extended accessibility features (high contrast and extended aria support)
```

> aria | null | Enable extended accessibility features (high contrast and extended aria support)

Reviewer 2: ```suggestion
aria | null | Enable extended accessibility features such as high contrast and extended aria support.
```

> created_at | null | The created_at date in your app for the end_user you are attempting to survey

Reviewer 2: ```suggestion
created_at | null | The created_at date in your app for the end_user you are attempting to survey.
```

email | null | The email of the end_user you are attempting to survey

> facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page

Reviewer 2: ```suggestion
facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page.
```

> language | null | The language code of the survey

Reviewer 2: ```suggestion
language | null | The language code of the survey.
```

product_name | depends on the language | Your product name. 

> properties  | null | The custom properties of the end_user you are attempting to survey

Reviewer 2: ```suggestion
properties  | null | The custom properties of the end_user you are attempting to survey.
```

> properties  | null | The custom properties of the end_user you are attempting to survey.

Reviewer 2: ```suggestion
properties | null | The custom properties of the end_user you are attempting to survey.
```

> twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment

Reviewer 2: ```suggestion
twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment.
```

> wootric_close_position | left | The position of the close button of your survey

Reviewer 2: ```suggestion
wootric_close_position | left | The position of the close button of your survey.
```

> wootric_recommend_target | depens of the language| Your audience

Reviewer 2: ```suggestion
wootric_recommend_target | depends of the language | Your audience
```

> wootric_recommend_target | depends of the language | Your audience

Reviewer 2: ```suggestion
wootric_recommend_target | depends of the language | Your audience.
```

> wootric_recommend_target | depends of the language | Your audience

Reviewer 2: ```suggestion
wootric_recommend_target | depends on the language | Your audience
```

> wootric_recommend_target | depends of the language | Your audience.

Reviewer 2: ```suggestion
wootric_recommend_target | depends on the language | Your audience.
```
